app/views/CreateChannelView.js

```diff
@@ -198,7 +208,7 @@ class CreateChannelView extends React.Component {
 			label: 'Private_Channel',
 			onValueChange: (value) => {
 				logEvent(events.CREATE_CHANNEL_TOGGLE_TYPE);
-				this.setState({ type: value });
+				this.setState(({ encrypted }) => ({ type: value, encrypted: value && encrypted }));
```

Comment:
I didn't get this

Reply:
If we set the channel as public, encrypted status should be false.
Added as a comment 👍